zephyr/patches.yml

```diff
@@ -132,3 +132,17 @@ patches:
     upstreamable: false
     comments: |
       Add k_busy_wait_ns() helper for nanosecond busy wait support.
+  - path: zephyr/rtt-logging-snippet.patch
```

Comment:
Can we make this a downstream snippet? The upstream PR has a change request, so I imagine we will be carrying this patch for a bit

Reply:
We might even just be able to make them permanent Kconfigs.